.github/workflows/test-and-upload-coverage.yml

```diff
@@ -35,31 +35,57 @@ jobs:
         database-type: [badger-file, badger-memory]
         mutation-type: [gql, collection-named, collection-save]
         lens-type: [wasm-time]
+        acp-type: [local]
         database-encryption: [false]
         include:
           - os: ubuntu-latest
             client-type: go
             database-type: badger-memory
             mutation-type: collection-save
             lens-type: wasm-time
+            acp-type: local
             database-encryption: true
           - os: ubuntu-latest
             client-type: go
             database-type: badger-memory
             mutation-type: collection-save
             lens-type: wazero
+            acp-type: local
             database-encryption: false
           - os: ubuntu-latest
             client-type: go
             database-type: badger-memory
             mutation-type: collection-save
             lens-type: wasmer
+            acp-type: local
+            database-encryption: false
+          - os: ubuntu-latest
+            client-type: go
+            database-type: badger-memory
+            mutation-type: collection-save
+            lens-type: wasm-time
+            acp-type: source-hub
+            database-encryption: false
+          - os: ubuntu-latest
+            client-type: http
+            database-type: badger-memory
+            mutation-type: collection-save
+            lens-type: wasm-time
+            acp-type: source-hub
+            database-encryption: false
+          - os: ubuntu-latest
+            client-type: cli
+            database-type: badger-memory
+            mutation-type: collection-save
+            lens-type: wasm-time
+            acp-type: source-hub
```

Comment:
suggestion: I think at this point we might as well just make another `job` in this same workflow file called `run-sourcehub-tests` with it's own strategy only running the three clients (`client-type: [go, http, cli]`). Will the extra `if: ${{ matrix.acp-type == 'source-hub' }}` conditions from this strategy. Then under the `needs: ...` of the `upload-coverage` job you can add another name `run-sourcehub-tests`

Reply:
I do prefer the current setup.  Everything works in the same way and is declared in the same location.  It is getting a bit lengthy, but is better than the alternative atm.